Framework/Algorithms/src/GeneratePythonScript.cpp

```diff
@@ -132,12 +138,14 @@ void GeneratePythonScript::exec() {
                         ignoreTheseAlgs, ignoreTheseAlgProperties,
                         appendExecCount);
   std::string generatedScript;
-  generatedScript += "#########################################################"
-                     "#############\n";
-  generatedScript +=
-      "#Python Script Generated by GeneratePythonScript Algorithm\n";
-  generatedScript += "#########################################################"
-                     "#############\n";
+  const std::string mantidVersion =
+      std::string(Mantid::Kernel::MantidVersion::version());
```

Comment:
I think some of these namespaces are declared with a `using` statement at the top of the file so
```suggestion
      std::string(MantidVersion::version());
```